src/behaviors/localization/locales/es.ts

```diff
@@ -24,7 +24,7 @@ const es:IPartialLocaleValues = {
         ],
         formats: {
             time: "HH:mm",
-            datetime: "D MMMM [de] YYYY HH:mm",
+            datetime: "D [de] MMMM [de] YYYY HH:mm",
```

Comment:
as @harogaston said, we - the ones from the _Río de la Plata_ - know better (?).